app/src/main/AndroidManifest.xml

```diff
@@ -30,7 +30,7 @@
             android:value="@string/google_maps_key" />
 
         <activity
-            android:name="com.example.conupods.MapsActivity"
+            android:name=".MapsActivity"
```

Comment:
shouldnt that be `com.conupods.MapsActivity`?

Reply:
hmm I think so too, but I didnt change this, the Android visual studio did, and it created the file